java/src/main/java/com/adhoc/flight/client/AdhocFlightClient.java

```diff
@@ -55,21 +63,26 @@
 
 import com.adhoc.flight.utils.QueryUtils;
 import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableMap;
 
 /**
  * Adhoc Flight Client encapsulating an active FlightClient and a corresponding
  * CredentialCallOption with a bearer token for subsequent FlightRPC requests.
  */
 public final class AdhocFlightClient implements AutoCloseable {
+  public static final String PROJECT_ID_KEY = "project_id";
```

Comment:
Should we expose this parameter out from the arrow flight client and not create it here?

Reply:
No. project_id is a Dremio concept and so won't be defined in the Arrow Flight client.